packages/vanilla/src/components/checkbox/indeterminate.js

```diff
@@ -0,0 +1,46 @@
+class IndeterminateCheckbox {
+  constructor(checkbox) {
+    this.indeterminateCheckbox = checkbox;
+    this.nestedCheckboxWrapper = this.indeterminateCheckbox.closest('.cbp-checkbox-item').nextElementSibling;
+    this.nestedCheckboxInputs = this.nestedCheckboxWrapper.querySelectorAll('input[type="checkbox"');
```

Comment:
I think you're missing the closing `]` in your attribute selector.

Reply:
thanks for catching that!